Lib/argparse.py

```diff
@@ -1016,7 +1016,8 @@ def __init__(self,
 
     def __call__(self, parser, namespace, values, option_string=None):
         items = getattr(namespace, self.dest, None)
-        items = _copy_items(items)
+        if items is self.default:
```

Comment:
As far as I can tell this changes the behavior in some corner cases. For example, when `parse_known_args()` is called with a not empty `namespace`. The new code will only clone lists when `default` is set. The old code would have cloned any existing list. This matters if the caller has another reference to the same list.

Something like this

```py
some_existing_list = []
ns = argparse.Namespace()
ns.setattr("foo", some_existing_list)

parser = argparse.ArgumentParser()
parser.add_argument('--foo', action='append')
...

... = parser.parse_known_args(args=['--foo=xxx'], namespace=ns)

# At this point some_existing_list is ['xxx']
```

Reply:
Good point. There are no tests for this (and some other corner cases).